### ISSUE TITLE
verify coco DLA training command

### ISSUE THREAD

**Issue by bsvineethiitg** [xingyizhou/CenterNet#264]
Hi. Can you please verify the training command you have put under experiments folder for COCO DLA 34 1x? Especially, is the LR 5e-4 on Adam and batch size = 128?

Because, I retrained the DLA 34 model on COCO (1x) and I get an extremely overfit result - not just in loss values, but also mAP (getting around 27 for model_last.pth at 140th epoch!). 

So, I wanted to double check if the parameters you've provided are correct. 

Thanks again!

**Comment by bsvineethiitg** [comment]
Glad if you can just confirm. Thanks.

**Comment by xingyizhou** [comment]
Yes it is correct. Please make sure you follow the linear learning rate rule if you changed the batch size.
